gulpfile.js

```diff
@@ -200,128 +200,128 @@ function clean_cache() {
+        './src/js/libraries/three/three.min.js',
+        './src/js/libraries/three/Projector.js',
+        './src/js/libraries/three/CanvasRenderer.js',
+        './src/js/libraries/jquery.flightindicators.js',
+        './src/js/libraries/semver.js',
+        './src/js/libraries/jbox/jBox.min.js',
+        './src/js/libraries/switchery/switchery.js',
+        './src/js/libraries/bluebird.min.js',
+        './src/js/libraries/jquery.ba-throttle-debounce.min.js',
+        './src/js/libraries/inflection.min.js',
+        './src/js/injected_methods.js',
+        './src/js/data_storage.js',
+        './src/js/workers/hex_parser.js',
+        './src/js/fc.js',
+        './src/js/port_handler.js',
+        './src/js/port_usage.js',
+        './src/js/serial.js',
+        './src/js/gui.js',
+        './src/js/huffman.js',
+        './src/js/default_huffman_tree.js',
+        './src/js/model.js',
+        './src/js/serial_backend.js',
+        './src/js/msp/MSPCodes.js',
+        './src/js/msp.js',
+        './src/js/msp/MSPHelper.js',
+        './src/js/backup_restore.js',
+        './src/js/peripherals.js',
+        './src/js/protocols/stm32.js',
+        './src/js/protocols/stm32usbdfu.js',
+        './src/js/localization.js',
+        './src/js/boards.js',
+        './src/js/RateCurve.js',
+        './src/js/Features.js',
+        './src/js/Beepers.js',
+        './src/js/release_checker.js',
+        './src/tabs/adjustments.js',
+        './src/tabs/auxiliary.js',
+        './src/tabs/cli.js',
+        './src/tabs/configuration.js',
+        './src/tabs/failsafe.js',
+        './src/tabs/firmware_flasher.js',
+        './src/tabs/gps.js',
+        './src/tabs/help.js',
+        './src/tabs/landing.js',
+        './src/tabs/led_strip.js',
+        './src/tabs/logging.js',
+        './src/tabs/map.js',
+        './src/tabs/motors.js',
+        './src/tabs/onboard_logging.js',
+        './src/tabs/osd.js',
+        './src/tabs/pid_tuning.js',
+        './src/tabs/ports.js',
+        './src/tabs/power.js',
+        './src/tabs/receiver.js',
+        './src/tabs/receiver_msp.js',
+        './src/tabs/sensors.js',
+        './src/tabs/servos.js',
+        './src/tabs/setup.js',
+        './src/tabs/setup_osd.js',
+        './src/tabs/transponder.js',
+        './src/main.js',
 
         // everything else
-        './package.json', // For NW.js
-        './manifest.json', // For Chrome app
-        './eventPage.js',
-        './*.html',
-        './tabs/*.html',
-        './images/**/*',
-        './_locales/**/*',
-        './css/font-awesome/fonts/*',
-        './css/opensans_webfontkit/*.{eot,svg,ttf,woff,woff2}',
-        './resources/*.json',
-        './resources/models/*',
-        './resources/osd/*.mcm',
-        './resources/motor_order/*.svg',
+        './src/eventPage.js',
+        './src/*.html',
+        './src/tabs/*.html',
+        './src/images/**/*',
+        './src/_locales/**/*',
```

Comment:
I think `_locales/` is another case that should be outside of the `src/` folder.

Reply:
Agreed, after a better thought - these are files that are maintained by other people (and maybe do not have the knowledge of nw.js apps) it is more logical to place this folder in the root.

Reply:
The `_locales/` folder? I agree on that - they pretty much a type of file on its own.

Reply:
I will also rename it to something like `./translations` and let gulp copy the files to `./dist/_locales`.